### ISSUE TITLE
haxe-gif

### ISSUE THREAD

**Issue by ibilon** [ianharrigan/openfl-installer#4]
Hi,

tried to run the app but I get this error:
`src/openfl/installer/custom/Animation.hx:34: characters 2-16 : gif.AnimatedGif has no field delayTime`
do you use a custom version of haxe-gif?

**Comment by ianharrigan** [comment]
No, it should be the latest... That said, haxeui itself now supports animated gifs using the yagp lib, so ill move to use that soon. 

Not sure what the error is about, but ill check it out shortly.

Cheers,
Ian
